demo/admin/src/dashboard/Dashboard.tsx

```diff
@@ -36,20 +38,28 @@ const Greeting = styled(Typography)`
     color: white;
 `;
 
+const meQuery = gql`
+    query Me {
```

Comment:
Das query muss eigentlich eindeutig benannt sein, weil du importierst ja GQLMeQuery aus einem zentralen graphql.generated. Wenn du exakt das gleiche query 2x brauchst musst du es irgendwo exportieren - was ich aber nicht machen würde.

Reply:
It's now currentUser and currentUserSignOut. In my opinion more logical and specific enough.

Reply:
You did misunderstand that :) (me is perfectly ok and used by many apis)

`query **Me** {` This part as to be unique, you need eg.

```
query DashboardCurrentUser {
    me {
    }
}
```

Reply:
I changed it to DashboardCurrentUser. However, I still like CurrentUser more than Me.